news_flash/geocode_extraction.py

```diff
@@ -0,0 +1,12 @@
+import googlemaps
+
+
+def geocode_extract(location, maps_key):
+    gmaps = googlemaps.Client(key=maps_key)
+    geocode_result = gmaps.geocode(location, language="iw", region="il")
+    lat = geocode_result[0]["geometry"]["location"]["lat"]
```

Comment:
if there are no results you'll get an error, is that OK?

Reply:
Fixed, returns None